applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -223,3 +237,45 @@ def MarkWakeTEElement(self):
                     elem.SetValue(CPFApp.KUTTA, False)
                 else: #Rest of elements touching the trailing edge but not part of the wake
                     elem.SetValue(CPFApp.WAKE, False)
+
+    def ExecuteFinalizeSolutionStep(self):
+        node_velocity_potential_above = 0
+        node_velocity_potential_below = 0
+
+        show_only_the_first_element = True
+        for elem in self.wake_model_part.Elements:
+            print('element: ',elem.Id)
+            for node in elem.GetNodes():
+                
+                # Compute the distance from the node to the trailing edge
+                x_distance_to_te = node.X - self.te.X
+                y_distance_to_te = node.Y - self.te.Y
+
+                # Compute the projection of the distance vector in the wake normal direction
+                distance_to_wake = x_distance_to_te*self.wake_normal[0] + \
+                    y_distance_to_te*self.wake_normal[1]
+                
+                # Nodes laying on the wake have a positive distance
+                if(abs(distance_to_wake) < self.epsilon):
+                    print('element: ',elem.Id,' has nodes that are laying on wake! CL may not be correct in all nodes!')
```

Comment:
```suggestion
                    KratosMultiphysics.Logger.PrintInfo('DefineWakeProcess2D', 'element: ',elem.Id,' has nodes that are laying on wake! CL may not be correct in all nodes!')
```
Please don't use `print` in final code, use the logger instead with the name of the class as first argument